Docker/arch/arch-vars.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
it would probably be easier if `-lts` were in the file name, and container image name.

Reply:
Fixed